rpm/xulrunner-qt5.spec

```
@@ -90,6 +90,7 @@ Patch41:    0041-sailfishos-gecko-Fix-gfxPlatform-AsyncPanZoomEnabled.patch
 Patch42:    0042-sailfishos-gecko-Avoid-incorrect-compiler-optimisati.patch
 Patch43:    0043-sailfishos-gecko-Drop-static-casting-from-BrowserChi.patch
 Patch44:    0044-sailfishos-docshell-Get-ContentFrameMessageManager-v.patch
+Patch45:    0001-Revert-Bug-1445570-Remove-EnsureEventualAfterPaint-t.patch
```

```suggestion
Patch45:    0045-Revert-Bug-1445570-Remove-EnsureEventualAfterPaint-t.patch
```
Rename patch to 0045